open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/static/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Egeria UI application",
   "author": "Egeria contributors",
   "license": "Apache-2.0",
-  "repository": "odpi/egrias",
+  "repository": "odpi/egria",
```

Comment:
is this a typo or intentional?

Reply:
typo! I fixed the only partial. Spotted the final 's' not the missing "e'